grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -661,10 +661,10 @@ class ApiController extends BaseController {
                             in = ParameterIn.PATH,
                             required = true,
                             description = "Profile id - UUID or Scientific name"),
-                    @Parameter(name = "imageId",
+                    @Parameter(name = "fileName",
                             in = ParameterIn.PATH,
                             required = true,
-                            description = "Image id - UUID"),
+                            description = "fileName"),
```

Comment:
make sure the value of fileName is assigned to parameter imageId since retrieveLocalThumbnailImage action still uses imageId